packages/paste-design-tokens/tokens/global/box-shadow.yml

```
@@ -231,10 +238,16 @@ props:
     value: "{!offset-0} {!offset-05} {!offset-0} {!palette-red-90}"
     comment: Stronger bottom shadow border for notification.
 
-  # Scrollable elements
-  shadow-scroll-inverse:
-    value: "{!offset-0} {!offset-50} {!offset-50} {!offset-0} {!palette-gray-100}"
-    comment: Shadow for scrollable elements on inverse elements.
-  shadow-scroll:
-    value: "{!offset-0} {!offset-20} {!offset-20} {!offset-0} {!palette-gray-100-transparent-20}"
-    comment: Shadow for scrollable elements.
+  # Left / Right
+  shadow-border-left:
+    value: "{!offset-20} {!offset-0} {!offset-20} -{!offset-20} {!palette-gray-100-transparent-20}"
```

```suggestion
    value: "{!offset-20} {!offset-0} {!offset-40} -{!offset-20} {!palette-gray-100-transparent-20}"
```

todo: Let's use offset-40 (8px) for the blur value for these 4 tokens. Looks a little more natural